ApiBundle/Controller/ContentController.php

```diff
@@ -113,7 +114,7 @@ public function listAction(Request $request)
         );
 
         $repository =  $this->get('open_orchestra_model.repository.content');
-        $contentCollection = $repository->findByContentTypeInLastVersionForPaginateAndSearch($contentType, $columnsNameToEntityAttribute, $columns, $search, $order, $skip, $limit);
+        $contentCollection = $repository->findByContentTypeInLastVersionForPaginateAndSearchAndSiteId($contentType, $columnsNameToEntityAttribute, $columns, $search, $siteId, $order, $skip, $limit);
```

Comment:
This naming is becoming totallly absurd :s
@itkg-alavieille dind'nt want to enumerate all the params so he suffixed it with 'ForPaginateAndSearch' instead. Now there's a random param added to the end ('AndSiteId'). Where is the consistency to talk about the SiteId and not about the other params ? (Without speaking of the ridiculous naming the method is going to have now :p)